sql/src/main/java/io/crate/lucene/LuceneQueryBuilder.java

```diff
@@ -456,11 +459,76 @@ public Query apply(Function input, Context context) throws IOException {
 
         class NotQuery implements FunctionToQuery {
 
+            class SymbolToNotNullContext {
```

Comment:
I made it private, but cannot be static since `NotQuery` is already inner class and not static.